androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/StopDetailsFilterPills.kt

```diff
@@ -53,6 +57,17 @@ fun StopDetailsFilterPills(
     onTapRoutePill: (PillFilter) -> Unit,
     onClearFilter: () -> Unit
 ) {
+
+    @Composable
+    fun pillHint(appliedFilter: StopDetailsFilter?, pillFilter: PillFilter): String {
+        val filteredToPill = appliedFilter?.routeId == pillFilter.id
+        return if (filteredToPill) {
+            stringResource(R.string.stop_filter_pills_remove_hint)
```

Comment:
Once we have combined stop + trip details, this state will not be possible. But in the meantime, we should represent it.